src/test/java/com/kaloyan/notetakingapp/service/AdminServiceUnitTests.java

```diff
@@ -0,0 +1,59 @@
+package com.kaloyan.notetakingapp.service;
+
+import com.kaloyan.notetakingapp.dto.UserDTO;
+import com.kaloyan.notetakingapp.model.Role;
+import com.kaloyan.notetakingapp.model.User;
+import com.kaloyan.notetakingapp.repository.UserRepository;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.nio.file.AccessDeniedException;
+import java.util.ArrayList;
+import java.util.UUID;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class AdminServiceUnitTests {
+
+    @MockBean
+    private UserRepository userRepository;
+
+    @InjectMocks
+    private AdminServiceImpl adminService;
+
+    private UUID userId;
+
+    private User user;
+
+    @BeforeAll
+    public void init(){
+        userId = UUID.randomUUID();
+        user = User.builder().id(userId).role(Role.ROLE_USER).notes(new ArrayList<>()).build();
+    }
+
```

Comment:
there is no test for deleteNote
all public methods must be tested - ideally 1 positive test and at least 1 (preferably more) negative test/s